includes/admin/beta-features.php

```diff
@@ -0,0 +1,316 @@
+
+			add_action( 'load-' . $page, array( $this, 'load' ) );
+			add_action( 'admin_enqueue_scripts', array( $this, 'localize_beta_features' ), 20 );
+		}
+
+		/**
+		 * Loads the admin beta features page.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function load() {
+			add_filter( 'admin_body_class', array( $this, 'admin_body_class' ) );
+			// disable filters (default to raw data)
+			acf_disable_filters();
+
+			// Include and register beta features before checking submit
+			$this->include_beta_features();
+
+			$this->check_submit();
+		}
+
+		/**
+		 * Modifies the admin body class.
+		 *
+		 * @since SCF 6.4.2
+		 *
+		 * @param string $classes Space-separated list of CSS classes.
+		 * @return string
+		 */
+		public function admin_body_class( $classes ) {
+			$classes .= ' acf-admin-page';
+			return $classes;
+		}
+
+		/**
+		 * Includes various beta feature-related files.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function include_beta_features() {
+			acf_include( 'includes/admin/beta-features/class-scf-beta-feature.php' );
+			acf_include( 'includes/admin/beta-features/class-scf-beta-feature-editor-sidebar.php' );
+
+			add_action( 'scf/include_admin_beta_features', array( $this, 'register_beta_features' ) );
+
+			do_action( 'scf/include_admin_beta_features' );
+		}
+
+		/**
+		 * Register default beta features.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function register_beta_features() {
+			scf_register_admin_beta_feature( 'SCF_Admin_Beta_Feature_Editor_Sidebar' );
+		}
+
+		/**
+		 * Verifies the nonces and submits the value if it passes.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function check_submit() {
+			// Check if form was submitted.
+			if ( ! isset( $_POST['scf_beta_features_nonce'] ) || ! wp_verify_nonce( sanitize_key( $_POST['scf_beta_features_nonce'] ), 'scf_beta_features_update' ) ) {
+				return;
+			}
+
+			$beta_features = $this->get_beta_features();
+			$updated       = false;
+
+			foreach ( $beta_features as $name => $beta_feature ) {
+				$enabled = isset( $_POST['scf_beta_features'][ $name ] ) && '1' === $_POST['scf_beta_features'][ $name ];
+				$beta_feature->set_enabled( $enabled );
+				$updated = true;
+			}
+
+			if ( $updated ) {
+				add_action( 'admin_notices', array( $this, 'admin_notices' ) );
+			}
+		}
+
+		/**
+		 * Display admin notices.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function admin_notices() {
+			?>
+			<div class="notice notice-success is-dismissible">
+				<p><?php esc_html_e( 'Beta feature settings updated successfully.', 'secure-custom-fields' ); ?></p>
+			</div>
+			<?php
+		}
+
+		/**
+		 * Admin Beta Features html
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function html() {
+			// vars
+			$screen = get_current_screen();
+			$active = acf_maybe_get_GET( 'beta_feature' );
```

Comment:
Do we need to save the `$active` value at the class level, considering it's only used inside this function?

Reply:
No, also, it is just a CSS feature, we can completely remove it. Nice catch.